src/payment/payment.ts

```diff
@@ -69,3 +69,15 @@ export interface ThreeDSecureToken {
 export interface HostedInstrument {
     shouldSaveInstrument?: boolean;
 }
+
+export interface BraintreePaypal {
```

Comment:
Is this specific to Braintree provider or any PayPal instrument in general?

Reply:
I'll check.

Reply:
Changed to `BraintreePaypalInstrument`